### ISSUE TITLE
Lab still has various issues, missing dependencies.

### ISSUE THREAD

**Issue by f3mshep** [learn-co-curriculum/react-rendering-lab#13]
Only the first two sections of the lab are tested for: (Circles and the review trend monitor)

Two errors that prevent the tests from passing: 

`````````````````````````````````````````````````````````````````
1st Error:

ReferenceError: Unknown plugin "transform-react-inline-elements" specified in "/home/alexandra/code/labs/react-rendering-lab-v-000/.babelrc" at 3, attempted to resolve relative to "/home/alexandra/code/labs/react-rendering-lab-v-000"
    at /home/alexandra/code/labs/react-rendering-lab-v-000/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17


Fix by running:
 npm install --save-dev babel-plugin-transform-react-inline-elements

```````````````````````````````````````````````````````````````````

2nd Error:

All tests were failing with this error once I added the babel plugin with this error message:

     Error: 
      Enzyme Internal Error: Enzyme expects an adapter to be configured, but found none. To
      configure an adapter, you should call `Enzyme.configure({ adapter: new Adapter() })`
      before using any of Enzyme's top level APIs, where `Adapter` is the adapter
      corresponding to the library currently being tested. For example:

      import Adapter from 'enzyme-adapter-react-15';

      To find out more about this, see http://airbnb.io/enzyme/docs/installation/index.html

Solved by putting the following in the tests/root file:

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

---------------------------------------------------------------------------------------------------------------------------------------------

The first error makes it look as though the student has done something wrong, such as missed a vital function, when it is not their fault. It could be very frustrating to debug, please look into this.

**Comment by realAndrewCohn** [comment]
the .babelrc file has been updated on the master branch and no longer errors when running `npm test`

**Comment by haeke** [comment]
Hello,

I'm posting this here because I have run into similar errors when trying to run the tests. Hopefully this will be useful for anyone who runs into the same issue. I was able to solve the issue by doing the following:

Error 1 - After running npm install and npm test I received the error below -

> Error: Couldn't find preset "es2015" relative to directory

I needed to add the babel-preset-es2015 and babel-preset-stage-0 versions below to the dependencies object in package.json

    "babel-preset-es2015": "^6.22.0",
    "babel-preset-stage-0": "^6.22.0",

After running npm install, I received the same error as f3mshep. If you follow the steps in part 2 above that will fix the error. From here I was able to run both npm test and learn commands with success.
